bash-preexec.sh

```diff
@@ -300,7 +300,7 @@ __bp_install() {
     local prior_trap
     # we can't easily do this with variable expansion. Leaving as sed command.
     # shellcheck disable=SC2001
-    prior_trap=$(sed "s/[^']*'\(.*\)'[^']*/\1/" <<<"${__bp_trap_string:-}")
+    prior_trap=$(builtin command sed "s/[^']*'\(.*\)'[^']*/\1/" <<<"${__bp_trap_string:-}")
```

Comment:
FWIW, we have been simply using `command` in `bash-completion` and `oh-my-bash`, though I know kitty shell integration prefixes `builtin` to everything where applicable.

Reply:
Yes, I made that as a last minute change given that I referenced `builtin` in my commit message. I think it's unlikely someone would attempt to redefine `command`, but using `builtin command` here does make this the most explicit.

Reply:
Yeah, either works for me, but then the next question would be whether we should prefix `builtin` to everything like `builtin local`, `builtin return`, `builtin unset`, `builtin set`, `builtin shopt`, etc.